sdk/core/src/http_client.rs

```diff
@@ -122,6 +127,19 @@ impl HttpClient for reqwest::Client {
     }
 }
 
+// wasm can not get the http version
```

Comment:
Why not?

Reply:
If I try to use it, I get:
```
PS C:\Users\cataggar\io\azure-sdk-for-rust> cargo check -p azure_core --target=wasm32-unknown-unknown
    Blocking waiting for file lock on build directory
    Checking azure_core v0.1.0 (C:\Users\cataggar\io\azure-sdk-for-rust\sdk\core)
error[E0599]: no method named `version` found for reference `&reqwest::Response` in the current scope
   --> sdk\core\src\http_client.rs:140:19
    |
140 |     Some(response.version())
    |                   ^^^^^^^ method not found in `&reqwest::Response`
 ```
 The code shows this:
 https://github.com/seanmonstar/reqwest/blob/1614c5ea64813ba24d268a2af22ab0d19b902bdf/src/wasm/response.rs#L67-L73
``` rust
    /* It might not be possible to detect this in JS?
    /// Get the HTTP `Version` of this `Response`.
    #[inline]
    pub fn version(&self) -> Version {
        self.http.version()
    }
    */
 ```